lte/gateway/python/magma/pipelined/main.py

```diff
@@ -223,6 +224,9 @@ def on_exit_server_thread():
         )
         collector.start()
 
+    notifier = sdnotify.SystemdNotifier()
+    notifier.notify("READY=1")
+
     # Run the service loop
     service.run()
 
```

Comment:
same as above pipelineD depends on directoryD.

I think it would be safe to add this notification for all python services. I do not see any downside.
Thanks.

Reply:
There does not seem to be a systemd dependency in orc8r/tools/ansible/roles/gateway_services/files/magma.service for mobilityd and directoryd. How is that handled?

Reply:
looks like bug to me.